core/arch/arm/sta/stats.c

```diff
@@ -38,6 +40,80 @@
 			{ 0x87, 0x94, 0x10, 0x02, 0xa5, 0xd5, 0xc6, 0x1b } }
 
 #define STATS_CMD_PAGER_STATS		0
+#define STATS_CMD_ALLOC_STATS		1
+
+#define STATS_NB_POOLS			3
+
+static TEE_Result get_alloc_stats(uint32_t type, TEE_Param p[4])
+{
+	struct tee_mm_pool_stats *stats;
+	uint32_t size_to_retrieve;
```

Comment:
`pool_id` and `size_to_retrieve` are parameters in and out, so are uint32_t. `i` scans pool_id.